src/plugins/kibana_react/public/field_button/field_button.tsx

```diff
@@ -100,7 +100,16 @@ export function FieldButton({
 
   return (
     <div className={classes} {...rest}>
-      <button onClick={onClick} {...buttonProps} className={buttonClasses}>
+      <button
+        onClick={(e) => {
+          if (e.type === 'click') {
```

Comment:
Curious, if it is ever something else than `'click'`?

Reply:
Ah, I see it can be `mouseenter`.

![image](https://user-images.githubusercontent.com/9773803/91961494-8630f500-ed0b-11ea-87ba-5c195e7520b0.png)

Reply:
yes, it could also be triggered by an enter key, but in this case it should already have focus